scripts/semver-checks.sh

```
@@ -2,4 +2,4 @@
 
 set -x
 
-cargo semver-checks -p scylla -p scylla-cql $@
+cargo semver-checks -p scylla -p scylla-cql -p scylla-cql-core $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)